docs/core/compatibility/cryptography.md

```diff
@@ -14,6 +14,7 @@ The following breaking changes are documented on this page:
 | [EnvelopedCms defaults to AES-256 encryption](#envelopedcms-defaults-to-aes-256-encryption) | 3.0 |
 | [Minimum size for RSAOpenSsl key generation has increased](#minimum-size-for-rsaopenssl-key-generation-has-increased) | 3.0 |
 | [.NET Core 3.0 prefers OpenSSL 1.1.x to OpenSSL 1.0.x](#net-core-30-prefers-openssl-11x-to-openssl-10x) | 3.0 |
+| [CryptoStream.Dispose transforms final block only when writing](#cryptostreamdispose-transforms-final-block-only-when-writing) | 3.0 |
 | [Boolean parameter of SignedCms.ComputeSignature is respected](#boolean-parameter-of-signedcmscomputesignature-is-respected) | 2.1 |
```

Comment:
Was it intended that the new section in the table is not the last one?

Reply:
I added as last one in 3.0 version since they seem to be sorted by that